test/common/upstream/ring_hash_lb_test.cc

```diff
@@ -623,9 +623,9 @@ TEST_P(RingHashLoadBalancerTest, HostWeightedLargeRing) {
     ++counts[port - 90];
   }
 
-  EXPECT_EQ(987, counts[0]);  // :90 | ~1000 expected hits
-  EXPECT_EQ(1932, counts[1]); // :91 | ~2000 expected hits
-  EXPECT_EQ(3081, counts[2]); // :92 | ~3000 expected hits
+  EXPECT_EQ(987, counts[0]);        // :90 | ~1000 expected hits
+  EXPECT_NEAR(1932, counts[1], 10); // :91 | ~2000 expected hits
```

Comment:
you can add code coverage with both runtime flag enabled & disabled. So you can keep this as `EXPECT_EQ` with runtime flag being disabled and add a new test case with runtime flag enabled.

Reply:
> you can add code coverage with both runtime flag enabled & disabled. So you can keep this as `EXPECT_EQ` with runtime flag being disabled and add a new test case with runtime flag enabled.

I did this change suggested. Thanks @suniltheta

Reply:
> If this changes the ring layout this can't be behind a runtime flag. I think it should be a configuration setting? That is where the documentation would end up.

Hi @mattklein123, I couldn't get code changes done for configuration settings. For now, I am uploading a version with runtime guarding enabled. Can you help me with the code changes for configuration setting? Thanks.

Reply:
Sorry I don't have time to help with this code. I would recommend looking through recent changes that add configuration for some examples.

Reply:
I will check @mattklein123. Thanks